modules/apps/auxano-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,390 @@
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+
+a {
+	color: #000;
+	font-family: "Helvetica Neue";
+	font-size: 12px;
+	font-weight: bold;
+	letter-spacing: 1.71px;
+	text-decoration: none;
+	text-transform: uppercase;
+}
+
+a:active {
+	text-decoration: none;
+}
+
+a:hover, #color-777:hover, .info-footer-container ul li a h3:hover, .new-arrivals-link:hover {
+	color: #BE9A79;
+	text-decoration: none;
+}
+
+a:visited {
+	border-bottom: 5px solid black;
+	text-decoration: underline;
+}
+
+a#sign-in {
+	margin: 27px 20px;
+}
+
+body {
+	background-color: #FFF1E4;
+}
+
+.card {
+	border-radius: 0;
+	border-width: none;
+	box-shadow: 0px 2px 4px #000;
+	display: flex;
+	flex: 1 1 0;
+	flex-direction: column;
+	height: auto;
+	margin: 20px;
+	min-width: 189px;
+}
+
+.card-container {
+	align-items: center;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-around;
+	margin: 20px;
+}
+
+#card-description {
+	display: flex;
+	flex-direction: column;
+	margin: 10px 10px 20px 10px;
+}
+
+#card-description h6 {
+	/* margin-bottom: 0px; */
+	margin-top: auto;
+}
+
+#card-description p {
+	margin: 10px 0px 20px 0px;
+}
+
+#colombia {
+	color: #FFF;
+	font-family: "Phosphate";
+	font-size: 60px;
+	letter-spacing: 4.8px;
+	/* margin: 0px; */
+	padding: 20px 40px 0px 40px;
+	text-align: center;
+}
+
+#colombia-region {
+	color: #FFF;
+	margin: 0px;
+	text-align: center;
+}
+
+#color-777 {
+	color: #777;
+}
+
+.container-fluid {
+	padding: 0px;
+}
+
+div#heading {
+	align-items: flex-start;
+	flex: 0 0 auto;
+}
+
+.email-subscription-container {
+	align-items: center;
+	background-color: #FFF;
+	display: flex;
+}
+
+#email-subscription-links {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	margin-bottom: 10px;
+	width: 100%;
+}
+
+.email-subscription-text {
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	margin: auto;
+	padding: 20px 0px 40px 0px;
+}
+
+.email-text-field {
+	border-color: #000;
+	display: block;
+	width: 70%;
+}
+
+footer#footer {
+	margin: 10px 20px;
+	padding: 0px 20px;
+}
+
+h1.site-title {
+	color: #000;
+	font-family: Utopia Std;
+	font-size: 30px;
+	letter-spacing: 2px;
+	margin: 0px 30px 0px 20px;
+	padding: 20px;
+}
+
+h2 {
+	font-family: Utopia Std;
+	font-size: 30px;
+	letter-spacing: 2px
+}
+
+h3 {
+	font-family: Helvetica Neue;
+	font-size: 12px;
+	font-weight: bold;
+	letter-spacing: 1.71px;
+	text-transform: uppercase;
+}
+
+h4 {
+	font-family: Utopia Std;
+	font-size: 16px;
+	font-weight: bold;
+	letter-spacing: 2.28px;
+	text-transform: uppercase;
+}
+
+h5 {
+	font-family: Utopia Std;
+	font-size: 12px;
+	font-weight: bold;
+	letter-spacing: 1.71px;
+	text-transform: uppercase;
+}
+
+h6 {
+	font-family: Helvetica Neue;
+	font-size: 10px;
+	font-weight: bold;
+	letter-spacing: 1.43px;
+	text-transform: uppercase;
+}
+
```

Comment:
Since all the headings use some similar CSS rules, we can SASS them all together and then make a CSS rule for the differences.

Ex.
```
``
h1, h2, h3 {
	font-family: Helvetica Neue;
	font-weight: bold;
	text-transform: uppercase;
}

h1 {
	font-size: 16px;
	letter-spacing: 1.12px;
}
```